app/models/pglogical_subscription.rb

```diff
@@ -101,15 +100,24 @@ def self.provider_node_attributes(node_name)
 
   private
 
-  def node_name
-    MiqPglogical::NODE_PREFIX + MiqRegion.my_region_number.to_s
+  def update_subscription
+    disable
+    provider_node_name = MiqPglogical.region_to_node_name(provider_region)
+    pglogical.node_dsn_update(provider_node_name, dsn)
+    enable
+  end
+
+  def create_subscription
+    create_node.check if !pglogical.enabled? || self.class.count == 0
+    id = "subscription_#{host.gsub(/\.|-/, "_")}"
+    pglogical.subscription_create(id, dsn, [MiqPglogical::REPLICATION_SET_NAME], false).check
```

Comment:
No, check is a method on `PG::Result` that raises if the result is in a bad state (http://www.rubydoc.info/gems/pg/PG/Result#check-instance_method)

I want to move that type of checking into the pglogical ar_extension in a followup.

I'll return `self` here for now.